File: Selenium/Activities/Activity13.py
from selenium import webdriver
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.firefox.service import Service as fireFoxService
from selenium.webdriver.common.by import By

service=fireFoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service)as driver:
    driver.get("https://training-support.net/selenium/tables")
    print("Page Title:- " + driver.title)

    row=driver.find_elements(By.XPATH,"//table[@class='ui celled striped table']//tr")
    cell=driver.find_elements(By.XPATH,"//table[@class='ui celled striped table']//tr//td")

    print("Row Count:- " , len(row))
    print("Cell Count:- ", len(cell))

    thrdRow=driver.find_elements(By.XPATH,"//table[@class='ui celled striped table']//tr[3]//td")
    print("Third row Value:- " , end=" ")
    for row in thrdRow:
        print( row.text ,end= " " + ",")

    SecondVellValue=driver.find_element(By.XPATH,"//table[@class='ui celled striped table']//tr[2]//td[2]")
    print('\n')
    print("Second Row Second Cell valuec:- " +SecondVellValue.text)

    driver.close()

File: Selenium/Activities/Activity13.java
package example;

import java.util.List;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity13 {

	public static void main(String[] args) {
		
		WebDriver driver = new ChromeDriver();
		driver.get("https://training-support.net/webelements/tables");
		System.out.println("Title of the page is: " + driver.getTitle());
		
		List<WebElement> row = driver.findElements(By.xpath("//table[contains(@class, 'w-full')]/tbody/tr"));
		System.out.println("Number Of Row's :- " +row.size());
		
		List<WebElement> col = driver.findElements(By.xpath("//table[contains(@class, 'w-full')]/thead/tr/th"));
		System.out.println("Number Of Col :- " +col.size());
		
		List<WebElement> Thirdrow = driver.findElements(By.xpath("//table[contains(@class, 'w-full')]/tbody/tr[3]/td"));
		
		for (WebElement webElement : Thirdrow) {
			System.out.println(webElement.getText());
			
		}
		
		 WebElement CellValue = driver.findElement(By.xpath("//table[contains(@class, 'w-full')]/tbody/tr[2]/td[2]"));
		 System.out.println("second row second column :-  " +CellValue.getText());
		 
		 driver.quit();
		
		

	}

}
